### DISCUSSION TITLE
I want to learn!!!!!🥺🥺

### DISCUSSION THREAD

**Discussion by Zelechos** [vercel/next.js#26293] [Help]
I want to learn how to contribute to repositories on github where to start learning this si My Github acount 
[My Github](https://github.com/Zelechos)

**Comment by jacobhq** [comment]
There are some good events like hacktoberfest that you could participate in. In hacktoberfest, you make pull requests and when they are merged, they give you a point. Once you get three points, you get a free t-shirt. Based on your GitHub profile, I'm assuming that you know what a pull request is.

It would also be an idea to follow topics you are interested in on your explore page, as then it will curate repos that you might be interested in.
